src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -179,6 +179,18 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundInterpolationOperation((BoundStringInsert)boundNode);
                 case BoundKind.LocalFunctionStatement:
                     return CreateBoundLocalFunctionStatementOperation((BoundLocalFunctionStatement)boundNode);
+                case BoundKind.ConstantPattern:
+                    return CreateBoundConstantPatternOperation((BoundConstantPattern)boundNode);
+                case BoundKind.DeclarationPattern:
+                    return CreateBoundDeclarationPatternOperation((BoundDeclarationPattern)boundNode);
+                case BoundKind.WildcardPattern:
+                    return null;
```

Comment:
This bound node is created for a default label within pattern switch statement. The bound pattern case clause which is its parent already exposes the label symbol for the default label. In the design meeting we decided that we will represent such a default label within pattern switch statement as a null pattern. See unit tests added for default label in pattern case clause for tree structure.